packages/shared/src/components/cards/list/WelcomePostList.tsx

```diff
@@ -128,7 +128,7 @@ export const WelcomePostList = forwardRef(function SharePostCard(
           onCommentClick={onCommentClick}
           onCopyLinkClick={onCopyLinkClick}
           onBookmarkClick={onBookmarkClick}
-          className="mt-4 laptop:mt-auto"
+          className={classNames('mt-4', image && 'laptop:mt-auto')}
```

Comment:
```suggestion
          className={classNames('mt-4', !!image && 'laptop:mt-auto')}
```
Maybe double safety in case we ever move it to a component 😅